### PR TITLE
Reduce log verbosity in Travis CI environment.

### PR DESCRIPTION
**Pull request by yifanyang**
Mocha and karma will not show logs for succeeded test cases in CI
environment. Local development still displays full logs.

The is to mitigate #535. The number of total log lines on Travis will drop by about 2/3 with this pull request.

### CODE REVIEW

File: config/mocha.browser.opts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+--timeout 20000
```
Comment (Feiyang1) [inline config/mocha.browser.opts]: should we add the google license as in the file you deleted?

File: config/ci.mocha.node.opts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+--reporter dot
+--exit
```
Comment (Feiyang1) [inline config/ci.mocha.node.opts]: can you please explain the 2 options? Is it possible to put comments in the file?

File: config/mocha.browser.opts (hunk 1 above)
Comment (yifanyang) [reply config/mocha.browser.opts]: I think only js/ts files require a license header? Js configuration files under the 'config' directory do have a license header, while all the others don't. But I'm not sure. Your call.

P.S. Mocha version needs an upgrade to support comments in mocha config file, if we decide to add a header.

File: config/ci.mocha.node.opts (hunk 2 above)
Comment (yifanyang) [reply config/ci.mocha.node.opts]: I think we can, although that requires us to upgrade mocha to the latest version (5.2.0) in all of our packages. We have 5.0.5 currently.

I can create another pull request to bump the mocha version in all of our SDK packages. This pull request will probably need approvals from all the SDK owners. Let me know if you want me to proceed with it.

**Pull request closed without merging**